splunk/splunk-operator
# CSPL-909: Add test cases to verify changes to CR spec are applied to clustered environments

**Test to verify changes to CR spec are applied to the pod, for Indexer and SH clusters environments**

### SVA Covered:
* C3 and M4 

### Test Scenario:
* Deploy the SVA 
* Update CPU Value in  CR Spec of Indexer Cluster and apply 
* Update CPU Value in CR Spec of Search Head Cluster and apply 
* Verify CPU is updated on Indexer Pods
* Verify CPU is updated on Search Head Pods

### Test Run 
![image](https://user-images.githubusercontent.com/9527069/112080870-f1bfa000-8b3f-11eb-9cc6-95f833d36f37.png)
